Add numba pinning

danschef opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

danschef committed:
Pin numba to avoid SegmentationFault.

danschef committed:
Update repo URL.

danschef committed:
Bump build number.

danschef commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

Commit:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.10.08.45.3

unbohn commented on README.md:
```diff
@@ -3,7 +3,7 @@ About sicor-feedstock
 Feedstock license: [BSD-3-Clause](https://github.com/conda-forge/sicor-feedstock/blob/main/LICENSE.txt)
 
-Home: https://git.gfz-potsdam.de/EnMAP/sicor
+Home: https://git.gfz.de.de/EnMAP/sicor
```
@danschef What happened to this URL? https://git.gfz.de.de/EnMAP/sicor doesn't exist. https://git.gfz-potsdam.de/EnMAP/sicor still seems to be working.

danschef replied:
The one is in the README is updated by re-rendering.

unbohn commented on recipe/meta.yaml:
```diff
@@ -83,7 +83,7 @@ test:
 
 about:
-  home: https://git.gfz-potsdam.de/EnMAP/sicor
+  home: https://git.gfz.de.de/EnMAP/sicor
```
Same here. See my comment above.

danschef replied:
Good catch, thanks!

danschef replied:
All the gfz-potsdam URLs will stop working next year due to the renaming of the GFZ. That´s why I replaced it.

unbohn replied:
Ah interesting! Haven't heard about the renaming. What's the new name?

danschef replied:
Yes, see [here](https://www.gfz.de/sektion/geodaetische-weltraumverfahren/ueberblick/detailseite-sektionsnews/gutes-neues-jahr-2025-mit-neuem-namen-fuer-das-gfz). 2025 was basically the transition year where the old and the new URL worked more or less in parallel. But in 2026, everything with "gfz-potsdam" will stop working, which means we need to update all URLs, emails, license headers, repo remotes, affiliations, .... It is quite fun.

conda-forge-admin commented:
Hi! This is the friendly conda-forge automerge bot!

I considered the following status checks when analyzing this PR:
 - **linter**: passed
 - **azure**: failed


Thus the PR was not passing and not merged.

danschef committed:
Fix URL.

conda-forge-admin commented:
Hi! This is the friendly conda-forge automerge bot!

Commits were made to this PR after the `automerge` label was added. For security reasons, I have disabled automerge by removing the `automerge` label. Please add the `automerge` label again (or ask a maintainer to do so) if you'd like to enable automerge again!

danschef commented:
@conda-forge-admin, please rerender

Commit:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.10.16.41.2

danschef committed:
Try with  --no-deps --no-build-isolation.

Added flags to pip install command to prevent dependency resolution and build isolation.

danschef committed:
Add missing entry points.

danschef committed:
Revert build isolation flag and remove python requirement from tests section.

danschef committed:
Update pip install command to use verbose mode

Added verbose flag to pip install command in meta.yaml.

danschef commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ `noarch: python` recipes should usually follow the syntax in our [documentation](https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python) for specifying the Python version.
   - For the `test.requires` section of the recipe, you should usually use the pin `python {{ python_min }}` for the `python` entry.
   - If the package requires a newer Python version than the currently supported minimum version on `conda-forge`, you can override the `python_min` variable by adding a Jinja2 `set` statement at the top of your recipe (or using an equivalent `context` variable for v1 recipes).


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/20129440045](https://github.com/conda-forge/conda-forge-webservices/actions/runs/20129440045). Examine the logs at this URL for more detail.</sub>

danschef commented:
Somehow, conda solves the test environment forever if `python {{ python_min }}` is present there. I just removed it for now.
